app/views/shared/_select_content_dialog.html.erb

```diff
@@ -251,6 +252,29 @@
       </div>
       <% end %>
       <% if !only || only.include?(:assignments) %>
+      <div id="cloned_assignments_select" class="module_item_option">
+        <div style="font-size: 0.8em;">
+          <i class="icon-assignment"></i>
+          <%= t :assignment_notice, 'Select the assignment you want to associate with this *module*, or add an assignment by selecting "New Assignment".', :wrapper => '<span class="holder_type">\1</span>' %>
+        </div>
+        <div style="margin-top: 5px;">
+        <select class="module_item_select" aria-label="Select the assignment you want to associate" multiple>
+          <% Course.find(1).assignment_groups.active.include_active_assignments.each do |group| %>
```

Comment:
We should probably create a global variable to hold the Content Library course ID instead of hardcoding 1 all over the place.  It will make it less confusing to read the code.